app/models/content_block.rb

```diff
@@ -0,0 +1,4 @@
+class ContentBlock < ActiveRecord::Base
+  include Sufia::ContentBlockBehavior
+  include Lerna::ContentBlockBehavior
```

Comment:
I'm on the fence as to whether we need a new module in Lerna for this, but I wouldn't hold up merging this PR because of it.